website/source/api/acl/acl.html.md

```diff
@@ -263,3 +263,112 @@ agent_prefix "" {
    policy = "read"
 }
 ```
+
+## Login to Auth Method
```

Comment:
Do we always want to abbreviate to "Auth Method"? How do others products talk about this, do they say auth method or authentication method? My biggest concern is consistency, but I'm just starting my review so maybe we are consistent.

Reply:
The name of the thing is "auth method" so I was sticking with style used on https://www.vaultproject.io/docs/auth/